ext/node/path.rs

```diff
@@ -0,0 +1,42 @@
+use std::path::Component;
+use std::path::PathBuf;
+
+/// Extenion to path_clean::PathClean
+pub trait PathClean<T> {
+  fn clean(&self) -> T;
+}
+
+impl PathClean<PathBuf> for PathBuf {
+  fn clean(&self) -> PathBuf {
+    path_clean(self)
+  }
+}
+
+fn path_clean(path: &PathBuf) -> PathBuf {
+  let path = path_clean::PathClean::clean(path);
+  if path.to_string_lossy().contains("..\\") {
+    // temporary workaround because path_clean::PathClean::clean is
+    // not good enough on windows
+    let mut components = Vec::new();
```

Comment:
I'm surprised this is enough to make tests pass - I was under the impression that we required it on Unix systems too. Maybe we should fork `path-clean` crate and apply the fix there?

Reply:
It's using path clean on both, but then does a check for `..\`, which would only happen on Windows.

Reply:
🤦 I missed the first line...